app/src/services/auth.services.ts

```diff
@@ -37,15 +54,15 @@ const GetSingleUserApplication = async (url: string): Promise<CCApp.Datum> => {
 
 const PostDelcineContentCreator = async (id: string): Promise<unknown> => {
   return await axios.put(
-    `http://127.0.0.1:8888/api/applications/${id}?action=reject`,
-    { data: { reason: "No" } }
+    `${BACKEND_URL}/api/applications/${id}?action=reject`,
+    { data: { reason: `No` } }
```

Comment:
Is this reason sufficient? Is it used to show something to developers or users?

Reply:
No idea.
It's onboardings code, we just fixed the hardcoded URL.
The (") insted of (') are an error autofix from a VS code extension.
We'll remove the extension in the future.